src/Validators/OptionalIfCheck.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using form_builder.Models;
+using form_builder.Models.Elements;
+
+namespace form_builder.Validators.IntegrityChecks.Form
```

Comment:
Has this been moved..? Just in the "Validators" folder. Should be in the "Validators/IntegrityChecks/Form"folder...?